FilterStatement.cfc

```diff
@@ -12,11 +12,11 @@ component accessors = "true" {
 		structAppend(
 			variables,
 			{
-				activeFieldList: arguments.queryable.getFieldList(),
-				parameters: [],
-				queryable: arguments.queryable,
-				where: "",
-				whereCriteria: []
+				"activeFieldList": arguments.queryable.getFieldList(),
```

Comment:
@timmixell is there a particular reason you're adding quotes? I haven't spotted where it's necessary, and our standard was to not do that unless necessary.

Reply:
hey @dsbrady - it's to preserve case during serialization - it's more for aesthetics during debugging than anything (at this point in time)